azure/placeholder-function-app-template.json

```diff
@@ -108,7 +112,7 @@
         },
         {
             "apiVersion": "2021-04-01",
-            "name": "app-service-plan",
+            "name": "[concat(variables('functionAppName'), '-app-service-plan-', parameters('utcValue'))]",

```

Comment:
this can just be [concat(variables('appServicePlanName'), '-', parameters('utcValue'))]

Reply:
the reference here https://github.com/SkillsFundingAgency/das-github-template/pull/20/files#diff-b2fa97626129b1dad79a16dcb55aa6f8fd443ab2708f13416e94d63cfd91ceecL241 also needs to be changed